addons/web/views/webclient_templates.xml

```diff
@@ -430,6 +430,7 @@
                 <script type="text/javascript" src="/web/static/tests/fields/field_utils_tests.js"></script>
                 <script type="text/javascript" src="/web/static/tests/fields/relational_fields_tests.js"></script>
                 <script type="text/javascript" src="/web/static/tests/fields/special_fields.js"></script>
+                <script type="text/javascript" src="/web/static/tests/fields/upgrade_fields_tests.js"></script>
```

Comment:
you'll need an enterprise commit to remove this file from the bundle (see how we do for upgrade_fields.js)